pkgs/development/libraries/LAStools/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, cmake }:
+stdenv.mkDerivation rec {
+
+  pname = "LAStools-${version}";
+  version = "201003";
```

Comment:
Might want to add a comment explaining the versioning

Reply:
fixed in 6ec587bea40c282a2afc77ef1c3fc5b0e66b3eb6